src/addressBook/index.js

```diff
@@ -381,6 +382,32 @@ export function createContact(
   };
 }
 
+// TODO: keeping this separate to maintain functionality but will likely adapt the old function to take a type if we need to reuse it for groups
+export async function findPersonContactInAddressBook(
```

Comment:
Move this into `src/models/contact/person/index.js` - we probably have to copy and rewrite some old functionality into other model modules as well (e.g. we should probably copy an equivalent version of `getProfiles` into `src/models/profile/index.js` - but make sure that it fits the limitations that we have for model modules).

We really should not add more dependencies on this old code module, we want to get rid of it as soon as we can.